### ISSUE TITLE
Robots.txt Is Now Missing Sitemap Fields

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#50]
# Description
```
Please provide a short description of what you have encountered below.
```
`robots.txt` supposedly list all `sitemap.xml` files. Right now, it's hidden so we need to fix it.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
`robots.txt` is working as normal.

# Current Behavior
```
Please specify the current behavior of software below.
```
`robots.txt` is not listing `sitemap.xml`.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
1) Boot up any site.
2) Go to `/robots.txt`.
3) Observe `sitemaps.xml` is missing.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Fixed in 5e21342bae9eff6bbc1bd4e0f08c61c78862c2f7

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
